src/api/rails.js

```diff
@@ -0,0 +1,7 @@
+import axios from 'axios';
```

Comment:
Great job importing axios on only 1 file

Reply:
Thank you @rvvergara

Reply:
Welcome!